funding/admin.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib import admin
+from simple_history.admin import SimpleHistoryAdmin
+
+from funding.models import FundingBody
+from funding.models import Attribution
+from funding.models import FundingSource
+from funding.models import Publication
+from funding.models import FundingSourceMembership
+
+# Register your models here.
+
+
+@admin.register(FundingBody)
+class FundingBodyAdmin(SimpleHistoryAdmin):
+    list_display = ('name', )
+
+
+@admin.register(Attribution)
+class AttributionAdmin(SimpleHistoryAdmin):
+    list_display = ('title', )
```

Comment:
As discussed, I think it may be beneficial to also display the 'created_by' and 'owner' fields as the current projects will contain an empty value for the 'title' field.